pkg/kapis/devops/v1alpha3/handler.go

```diff
@@ -255,6 +255,7 @@ func (h *devopsHandler) DeletePipeline(request *restful.Request, response *restf
 	devops := request.PathParameter("devops")
 	pipeline := request.PathParameter("pipeline")
 
+	klog.V(7).Infof("ready to delete pipeline %s/%s", devops, pipeline)
```

Comment:
It uses level 8 in DevOps.

In addition, we may need to change it to a configurable variable.

Reply:
Got it. I can change it to 8 for now. Maybe we can put such specifications into a document. So everyone can follow it.